dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -867,6 +877,20 @@ public void testGettingImagesFromQuay() {
         WorkflowVersion versionWithDuplicateImages = snapshotWorkflowVersion(workflowsApi, "dockstore-testing/zhanghj-8555114", "/main.cwl", "1.0");
         assertEquals("Should have grabbed 3 images", 3, versionWithDuplicateImages.getImages().size());
         verifyChecksumsAreSaved(versionWithDuplicateImages);
+        versions = ga4Ghv20Api.toolsIdVersionsGet("#workflow/github.com/dockstore-testing/zhanghj-8555114");
+        testTRSConversion(versions, "1.0", 3);
+    }
+
+    private void testTRSConversion(final List<ToolVersion> versions, String snapShottedVersionName, final int numImages) {
+        for (ToolVersion trsVersion : versions) {
```

Comment:
Will pass if something horribly breaks and there are no versions, should have an assert that this is working as expected and that there are versions (and if I understand the intent, that that one of them should be the snapShottedVersionName)